common-utils/src/common/deployment/__init__.py

```diff
@@ -28,37 +24,25 @@
     PrivateAttr,
     StrictStr,
     conint,
-    validator,
 )
 from pydantic.error_wrappers import ValidationError
 from slugify import slugify
 
-# Create logger
+# Create logger for module
 LOGGER_NAME = __name__
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 # script path for referencing bash scripts
 SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
 
-# environment config for CI/CD
-ENVIRONMENT_TYPE = os.getenv("PREFECT_ENVIRONMENT_TYPE", "dev").lower()
```

Comment:
removing ENVIRONMENT_TYPE logic in favor of only DEPLOYMENT_TYPE which makes thing much more simple and less verbose